docs/samples/Microsoft.ML.Samples/Dynamic/KeyToValue_Term.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+        // the alignment of the usings with the methods is intentional so they can display on the same level in the docs site.
+        using Microsoft.ML.Data;
+        using Microsoft.ML.Runtime.Api;
+        using Microsoft.ML.Runtime.Data;
+        using Microsoft.ML.Transforms;
+        using System;
+        using System.Collections.Generic;
+        using System.Linq;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public partial class TransformSamples
+    {
+        public static void KeyToValue_Term()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext(seed: 1, conc: 1);
+
+            // Get a small dataset as an IEnumerable.
+            IEnumerable<SamplesUtils.DatasetUtils.SampleTopicsData> data = SamplesUtils.DatasetUtils.GetTopicsData();
+            var trainData = ml.CreateStreamingDataView(data);
+
+            // Preview of the data.
+            // Review                                    ReviewReverse,                 Label
+            // "animals birds cats dogs fish horse", "radiation galaxy universe duck",     1
+            // "horse birds house fish duck cats",   "space galaxy universe radiation",    0
+            // "car truck driver bus pickup",        "bus pickup",                         1
+            // "car truck driver bus pickup horse",  "car truck",                          0
+
+            // A pipeline to convert the terms of the review_reverse column in 
+            // making use of default settings.
+            string defaultColumnName = "DefaultKeys";
+            // REVIEW create through the catalog extension
+            var default_pipeline = new WordTokenizer(ml, "ReviewReverse", "ReviewReverse")
+                .Append(new TermEstimator(ml, "ReviewReverse" , defaultColumnName));
+
+            // Another pipeline, that customizes the advanced settings of the TermEstimator.
```

Comment:
May want to say _why_ changing the hyperparameters of the TermEstimator is useful. Why keep only the first 10 ASCIIbetically ordered terms?

Reply:
see if you'd phrase it differently. Maybe we can leave longer explanations for the documentation?

---
In reply to: [227236905](https://github.com/dotnet/machinelearning/pull/1321#discussion_r227236905) [](ancestors = 227236905)